### PR TITLE
Feat: Make localization work with arguments

### PR DESCRIPTION
**Pull request by mat1th**
When there is a localization key with arguments the `ParleyLocalizationManager` is not used, but `.localizedStringWithFormat` instead. This will move that also to the `ParleyLocalizationManager`.

### CODE REVIEW

**Comment by mat1th** [comment]
@alexkok When you have merged this could you create a new release? So we can use it directly?

File: Sources/Parley/Views/ParleyMessageView/ParleyMessageView.swift
Comment (alexkok) [inline Sources/Parley/Views/ParleyMessageView/ParleyMessageView.swift:592]: if goes into strange format

Could be something like 
```
   if !message.hasFile &&
      (message.title != nil || displayName == .message || message.message != nil || message.hasImage)
```

**Review by alexkok — APPROVED** [review]

**Comment by alexkok** [comment]
Thanks @mat1th ! Sure, will make a new release after merging.
Can you fix the format comment? Then I'll merge and release right after 👍

**Comment by alexkok** [comment]
Released in [4.2.1](https://github.com/parley-messaging/ios-library/releases/tag/4.2.1) 🚀

**Pull request merged**
